### PR TITLE
Test bump of recipes to 0.2.0

### PR DESCRIPTION
**Pull request by hayesgb**
This bumps the version of many of the packages in the coiled-runtime, and adds Optuna.

EDIT: Removing blosc as we drop support in https://github.com/dask/distributed/pull/6027/files

### CODE REVIEW

**Comment by crusaderky** [comment]
You should pin bokeh too (https://github.com/dask/distributed/pull/7219)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,34 +23,35 @@ requirements:
...
-    - lz4 ==4.0.0
-    - ipywidgets ==7.7.0
+    - fsspec ==2022.10.0
+    - s3fs ==2022.10.0
+    - gcsfs ==2022.10.0
+    - pyarrow ==9.0.0
+    - jupyterlab ==3.4.8
+    - dask-labextension ==5.3.0
+    - lz4 ==4.0.2
+    - ipywidgets ==8.0.2
```
Comment (crusaderky) [inline recipe/meta.yaml:37]:
```suggestion
    - ipywidgets ==7.7.2
```
Coiled requires <8.0: https://github.com/conda-forge/coiled-feedstock/blob/main/recipe/meta.yaml

File: recipe/meta.yaml [hunk 2]
```diff
@@ -22,35 +22,36 @@ requirements:
...
-    - python-blosc ==1.10.2
+    - toolz ==0.12.0
+    - python-blosc ==1.10.6
     - zict ==2.2.0
-    - xgboost ==1.6.1
+    - xgboost ==1.6.2
     - dask-ml ==2022.5.27
     - openssl >1.1.0g
     - pynvml ==11.4.1
+    - optuna ==3.0.3
```
Comment (crusaderky) [inline recipe/meta.yaml:54]:
```suggestion
    - bokeh ==2.4.3
```

Comment (ncclementi) [reply recipe/meta.yaml:54]: Thanks, @crusaderky. I will commit it when this first CI run finishes.

**Comment by ncclementi** [comment]
We have 3 errors that I'm not sure what's causing them, 
```python-traceback
 ERROR tests/benchmarks/h2o/test_h2o_benchmarks.py::test_q8[0.5 GB (parquet)] - asyncio.exceptions.TimeoutError: Waited for 10 worker(s) to reconnect after restarting, but after 120s, only 9 have returned. Consider a longer timeout, or `wait_for_workers=False`.
ERROR tests/benchmarks/test_array.py::test_map_overlap_sample - asyncio.exceptions.TimeoutError: Waited for 10 worker(s) to reconnect after restarting, but after 120s, only 9 have returned. Consider a longer timeout, or `wait_for_workers=False`.
ERROR tests/stability/test_shuffle.py::test_shuffle_parquet - asyncio.exceptions.TimeoutError: Waited for 10 worker(s) to reconnect after restarting, but after 120s, only 9 have returned. Consider a longer timeout, or `wait_for_workers=False`.
```

@crusaderky any thoughts?

**Comment by crusaderky** [comment]
> @crusaderky any thoughts?

Nothing from the top of my head. Are there logs?

**Comment by ncclementi** [comment]
Doesn't look like it https://github.com/coiled/coiled-runtime/actions/runs/3389153565/jobs/5632039514

**Comment by ncclementi** [comment]
I re-run things and now I see 
```
 ERROR tests/benchmarks/test_array.py::test_basic_sum[slow-square] - distributed.comm.core.CommClosedError: in <TLS (closed) ConnectionPool.run_function local=tls://10.1.0.18:39092 remote=tls://3.145.166.7:8786>: TimeoutError: [Errno 110] Connection timed out
```
again I don't see logs in the CI run. I've seen this over here https://github.com/coiled/coiled-runtime/issues/460
any thoughts? @ntabris @crusaderky

**Comment by ntabris** [comment]
> any thoughts?

sorry, what's the question / specific issue you'd like help investigating?

**Comment by ncclementi** [comment]
> sorry, what's the question / specific issue you'd like help investigating?

@ntabris I've seen this error showing multiple times, causing CI to fail, do we know the nature of it?
It happened here in  https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:366
and over https://github.com/coiled/coiled-runtime/actions/runs/3283399001/jobs/5408063717#step:5:272

**Comment by ncclementi** [comment]
Things are green now, but we should keep an eye on the failures mentioned here https://github.com/coiled/coiled-runtime/pull/488#issuecomment-1303711513
they are unrelated, but worth looking at them. 

I'll merge this in and test performance for a few days, and next week we can get a release out.

**Review by ncclementi — APPROVED** [review]
LGTM thanks @hayesgb

**Comment by ntabris** [comment]
the error in https://github.com/coiled/coiled-runtime/pull/488#issuecomment-1303711513 seems like a dask problem to me:

```
self = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0>
[282](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:283)
len = 65536
[283](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:284)
buffer = bytearray(b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x...0\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')
[284](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:285)

[285](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:286)
    def read(self, len=1024, buffer=None):
[286](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:287)
        """Read up to LEN bytes and return them.
[287](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:288)
        Return zero-length string on EOF."""
[288](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:289)
    
[289](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:290)
        self._checkClosed()
[290](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:291)
        if self._sslobj is None:
[291](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:292)
            raise ValueError("Read on closed or unwrapped SSL socket.")
[292](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:293)
        try:
[293](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:294)
            if buffer is not None:
[294](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:295)
>               return self._sslobj.read(len, buffer)
[295](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:296)
E               TimeoutError: [Errno 110] Connection timed out
[296](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:297)

[297](https://github.com/coiled/coiled-runtime/actions/runs/3394037683/jobs/5642091376#step:5:298)
/usr/share/miniconda3/envs/test/lib/python3.9/ssl.py:1100: TimeoutError
```

**Pull request merged**